codec-http2/src/main/java/io/netty/handler/codec/http2/Http2SecurityUtil.java

```diff
@@ -34,26 +34,48 @@
      * href="https://wiki.mozilla.org/Security/Server_Side_TLS#Non-Backward_Compatible_Ciphersuite">Mozilla Cipher
      * Suites</a> in accordance with the <a
      * href="https://tools.ietf.org/html/draft-ietf-httpbis-http2-16#section-9.2.2">HTTP/2 Specification</a>.
+     * 
+     * On OpenJDK, the ciphers start with TLS_, while on IBM J9 they are named SSL_.
```

Comment:
I think its better to just list that the JSSE allows for the "TLS" and "SSL" prefix to be interchanged [1]:

> The names mentioned in the TLS RFCs prefixed with TLS_ are functionally equivalent to the JSSE cipher suites prefixed with SSL_.

[1] http://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html